src/schema/v2/homeView/sections/Tasks.ts

```diff
@@ -0,0 +1,40 @@
+import { connectionFromArraySlice } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { createPageCursors } from "schema/v2/fields/pagination"
+import { HomeViewSection } from "."
+import { withHomeViewTimeout } from "../helpers/withHomeViewTimeout"
+import { HomeViewSectionTypeNames } from "../sectionTypes/names"
+
+export const Tasks: HomeViewSection = {
+  id: "home-view-section-tasks",
+  type: HomeViewSectionTypeNames.HomeViewSectionTasks,
+  // TODO: Create context module in Cohesion
+  // contextModule: ContextModule.tasks,
+  component: {
+    title: "Notifications",
```

Comment:
I know this isn't the best place for product feedback but do we think "Notification" too easily conflicts with "Latest Activity"? Have we considered using "Tasks" or something similar instead?

Reply:
<img width="300" alt="signal-2024-10-10-115015_002" src="https://github.com/user-attachments/assets/ecb8d010-0268-4225-a839-e22b6687738e" />
<img width="300" alt="signal-2024-10-10-115015_003" src="https://github.com/user-attachments/assets/8c384cf3-e72d-4410-aa3a-443276d2ca83" />

Reply:
<img width="622" alt="Screenshot 2024-10-10 at 15 39 01" src="https://github.com/user-attachments/assets/2d77694e-936c-4571-8237-6baab90966e4">